apps/datacalc.py

```diff
@@ -119,6 +119,11 @@ def setDB(self):
         """Sets the databases to fetch the numbers."""
         for name, dbBox in self.viewerFrame.dbBoxes.items():
             self.dbNames[name] = dbBox.currentText()
+            self.broadcastRequested.emit(
+                "logbus", 
+                f"Database {name} is set as {self.dbNames[name]}." if self.dbNames[name]
+                else f"Database {name} is not selected."
```

Comment:
I think `if` and `else` are too far, so it is not comfortable to read.
I'd prefer to break the line before `if`, i.e.,
```python
f"..."
if self.dbNames[name]
else f"..."
```

Reply:
Okay! I was also thinking of this part.